composer.json

```diff
@@ -30,7 +30,7 @@
   },
   "require": {
     "php": ">=7.1",
-    "infinum/eightshift-libs": "^2.1.1"
+    "infinum/eightshift-libs": "^2.2.0"
```

Comment:
please remove this I will add this changes when I create a release. and also from the changelog

Reply:
But the change cannot be merged before libs are updated 😬 If you were to merge it now, it would be broken. I'd rather leave it in until the new release is out (and I added it as a 2.2.0 since it's a bit of a bc break)